content/rest/channels.textile

```diff
@@ -274,9 +293,15 @@ To publish to a channel, make use of the "publish":#publish method of the channe
 ```
 
 ```[go]
-  rest, err := ably.NewRestClient(ably.NewClientOptions("{{API_KEY}}"))
+  rest, err := ably.NewREST(ably.WithKey("{{API_KEY}}"))
+  if err != nil {
+      panic(err)
+  }
   channel := rest.Channels.Get("{{RANDOM_CHANNEL_NAME}}")
-  err = channel.Publish("example", "message data")
+  channel.Publish(context.Background(), "example", "message data")
```

Comment:
This `channel.Publish` is hanging on for dear life 😅

Reply:
How embarrassing...clearly a Monday! 🤦